theseus_gui/src/components/ui/Instance.vue

```diff
@@ -21,6 +26,27 @@ const props = defineProps({
       </div>
       <div class="cta"><PlusIcon /></div>
     </Card>
+    <ProjectCard
+      v-else-if="display === 'project'"
+      class="instance-project-item"
+      :id="props.instance?.slug"
+      :type="props.instance?.project_type"
+      :name="props.instance?.title"
+      :description="props.instance?.description"
+      :iconUrl="props.instance?.icon_url"
+      :downloads="props.instance?.downloads?.toString()"
+      :follows="props.instance?.follows"
+      :createdAt="props.instance?.date_created"
+      :updatedAt="props.instance?.date_modified"
+      :categories="getCategoriesByInstanceId(props.instance?.project_id)"
+      :projectTypeDisplay="props.instance?.project_type"
+      projectTypeUrl="mod"
+      :serverSide="props.instance?.server_side"
+      :clientSide="props.instance?.client_side"
+      :showUpdatedDate="false"
+      :color="props.instance?.color"
+    >
+    </ProjectCard>
```

Comment:
This feels weird to be part of the the "Instance" component/encapsulating ProjectCard like this. I think it's better to just directly use ProjectCard component when needed